namelist.canopy

```diff
@@ -1,7 +1,7 @@
 &FILENAMES
   file_vars    = 'gfs.t12z.20220701.sfcf000.nc'
 !  file_vars    = 'gfs.t12z.20220701.sfcf000.txt'
-  file_out     = 'test_out'
+  file_out     = 'test_out_se'
```

Comment:
'se'?

Reply:
This just refers to southeast us example.  Will update it to be clearer.